packages/kit/.gitignore

```diff
@@ -8,4 +8,8 @@
 !/src/vite/build/adapt/fixtures/*/.svelte-kit
 !/test/node_modules
 /test/apps/basics/test/errors.json
-.custom-out-dir
\ No newline at end of file
+.custom-out-dir
+
+# these are already ignored by the top level .gitignore
+# repeating them here as a faux prettier ignore
+/src/packaging/test/**/.svelte-kit
```

Comment:
we also have `/test/**/.svelte-kit` above. I wonder if we should combine the two into something like `**/.svelte-kit`?

Reply:
The `**/` can probably be omitted as well, the top level gitignore already does so.